lib/vmdb/logging.rb

```diff
@@ -1,7 +1,18 @@
 module Vmdb
   class LogProxy < Struct.new(:klass, :separator)
+    class NullLogger < Logger
+      def initialize(*args)
+        super('/dev/null')
+        self.level = Logger::UNKNOWN
+      end
+
+      def log_backtrace(*_); end
+      def success(*_); end
+      def failure(*_); end
```

Comment:
Lose the underscore on these; it's not needed, and will cause an array allocation